bintray.json

```diff
@@ -1,15 +1,15 @@
 {
     "package": {
-        "name": "OpenMRS-Add-On-Manager",
```

Comment:
The package should be openmrs-owa-addonmanager

Reply:
Fixed